frontend/src/containers/pages/PlayScenarioPage/PlayScenarioPage.jsx

```diff
@@ -39,37 +39,33 @@ export default function PlayScenarioPage() {
   const { scenarioId, sceneId } = useParams();
   const history = useHistory();
   const styles = useStyles();
-
   const [previous, setPrevious] = useState(null);
-  const [error, setError] = useState(null);
 
-  // TODO: move these somewhere else ?
-  if (loading) return <LoadingPage text="Loading Scene..." />;
-  if (authError) return <></>;
+  const setError = useCallback((error) => {
```

Comment:
nit: naming suggests it is a setter function, but it doesn't set any state _and_ has side effects

Reply:
`handleError` may be more appropriate?